app/views/components/_datetime_fields.html.erb

```diff
@@ -1,6 +1,7 @@
 <%
  prefix = prefix
  field_name = field_name
```

Comment:
I think this can be removed lol

Reply:
I think `field_name` needs to be there otherwise `field_name: "#{field_name}(3i)"` causes an error.

Reply:
I've left these as they are.